### DISCUSSION TITLE
OpenID integration with DEX inside k8

### DISCUSSION THREAD

**Discussion by Marwan-Dalaty** [allure-framework/allure2#2399] [TestOps]
Dear support, 

In your documentation you just mentioned OIDC with Okta, keycloak and Azure. I'm trying to connect OIDC with DEX OpenID. Eveyrthing is working fine when it comes to integration and I'm able to authenticate and login but I cant manage the groups permission like to map the groups permission between github and allure roles. 
```
      syncRoles: false
      # Maps your OIDC groups on Allure Groups
      userGroupName: allure_users
      adminGroupName: allure_admins

```
Can you advise please

**Comment by cheshi-mantu** [comment]
Hi @Marwan-Dalaty could we proceed in https://help.qameta.io (we have dedicated support for Allure Testops).
Please user the email address of the company which has the licence.
cheers
